SPECS/linstor-client.spec

```diff
@@ -0,0 +1,43 @@
+Summary: DRBD distributed resource management utility
+Name:    linstor-client
+Version: 1.19.0
+Release: 1%{?dist}
+License: GPLv3
```

Comment:
Our existing packages are not a very good example, but now we try to use valid SPDX identifiers for the License tag. This would be either `GPL-3.0-only` or `GPL-3.0-or-later`.

Reply:
I picked the same license identifier as in the linbit packages so I cannot tell which one is best. And the linbit github repo doesn't help either.

Reply:
Fixed in new separated PRs for [8.2](https://github.com/xcp-ng-rpms/linstor-client/pull/2) and [8.3](https://github.com/xcp-ng-rpms/linstor-client/pull/3)